scripts/pylint_all.py

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env python3
+
+"""
+Performs pylint on all python files in the project repo's test directory recursively.
```

Comment:
This comment is not correct as it also runs for `script/`. Also there is one python script in `docs/` that doesn't seem to be tested?

Reply:
it still says "in the repo's test directory"?